src/main/java/pl/targosz/homework/CodeRefactoringLesson5Task5/Bicycle.java

```diff
@@ -0,0 +1,23 @@
+package pl.targosz.homework.CodeRefactoringLesson5Task5;
+
+public class Bicycle extends Vehicle {
+
+    protected void startEngine() {
+        throw new UnsupportedOperationException("Bicycle does not have engine");
```

Comment:
skoro wymagamy, aby bicycle implementowało metodę startEngine/stopEngine mimo, że nie ma silnika to znaczy, że coś tu jest nie tak i wymaga refactoru.